messaging/WFLY-5286_add_runtime_journal_type_attribute.adoc

```diff
@@ -0,0 +1,71 @@
+= <INSERT TITLE HERE>
+:author:            Emmanuel hugonnet
+:email:             ehugonne@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:          messaging,jms
+:idseparator:       -
+
+== Overview
+
+When configuring the use of AIO for journal persistence, the Artemis server may fall back to NIO if the libaio binaries are not located. This state is not displayed to the user so he doesn't know the effective journal type that is used.
+
```

Comment:
I think new `runtime-journal-type` attribute should be also mentioned in an overview so it was clear that this is how the issue is going to be solved.